app/serializers/axiom_serializer.rb

```diff
@@ -0,0 +1,23 @@
+class AxiomSerializer < ApplicationSerializer
+  class Reference < ApplicationSerializer
+    attributes :iri
+    attributes :name
+
+    def iri
+      qualified_locid_for(object)
+    end
+  end
```

Comment:
good. Could this explanation please be added in the code (or the doc) somewhere?